docs/authentication.md

```diff
@@ -0,0 +1,318 @@
+---
+description: Learn about authentication patterns in Next.js apps and explore a few examples.
+---
+
+# Authentication
+
+Authentication verifies who a user is, while authorization controls what a user can access. Next.js supports multiple patterns for authentication, each designed for different use cases. This guide will allow you to choose your adventure based on your constraints.
```

Comment:
> Next.js supports multiple patterns for authentication, each designed for different use cases. This guide will allow you to choose your adventure based on your constraints.

This sounds like if it were a Next.js feature 🤔

Reply:
Yeah, that's a good point. In a way, it kind of is. Using CRA you wouldn't be able to support SSR authentication. Would this read better with the last sentence removed?